README.md

```diff
@@ -98,7 +98,47 @@ payloads.
 />
 ```
 
-### OAuth redirects
+### Widget configuration props
+
+You can configure the state and behaviour of the widget with the following
+component props:
+
+- `language`: Load the widget in the specified language. Defaults to `en-US`.
+  See [language
+  options](https://docs.mx.com/api#connect_configuring_connect_language_options)
+  for additional information.
+
+#### Widget configuration props for Connect
```

Comment:
> #### Widget configuration props for Connect

\- or -

> #### Connect specific configuration props

Reply:
I think I prefer the second, but not a big deal.

Reply:
Yeah, same. Just updated it.